### PR TITLE
implemented lambda failure and runid

### PR DESCRIPTION
**Pull request by kingmushroom**

### CODE REVIEW

File: strata_period_wrangler.py [hunk 1]
```diff
@@ -190,11 +199,12 @@ def lambda_handler(event, context):
                 + str(e.args)
                 + " | Request ID: "
                 + str(context.aws_request_id)
+                + " | Run_id: " + str(run_id)
         )
         log_message = error_message + " | Line: " + str(e.__traceback__.tb_lineno)
     except exception_classes.MethodFailure as e:
         error_message = e.error_message
-        log_message = "Error in " + method_name + "."
+        log_message = "Error in " + method_name + "." + " | Run_id: " + str(run_id)
```
Comment (Jordancooke) [inline strata_period_wrangler.py:207]:
```
"." + " | Run_id: "
```
Possibly could move the . inside the following string like so: ". | Run_id ".

**Review by Jordancooke — APPROVED** [review]
Only suggestion I have is a possible nice to have so approved :)

File: strata_period_wrangler.py [hunk 2]
```diff
@@ -45,8 +45,13 @@ def lambda_handler(event, context):
     logger = logging.getLogger("Strata")
     logger.setLevel(10)
 
+    # Define run_id outside of try block
+    run_id = 0
     try:
         logger.info("Strata Wrangler Begun")
+        # Retrieve run_id before input validation
+        # Because it is used in exception handling
+        run_id = event['RuntimeVariables']['id']
```
Comment (Piwington) [inline strata_period_wrangler.py]: Didn't we decide upon 'run_id' over 'id' in the end?

Comment (kingmushroom) [reply strata_period_wrangler.py]: quite right, i'll get them altered before starting enrichment.

**Review by Piwington — APPROVED** [review]

**Pull request merged**